### PR TITLE
[RDY] Rewrite English man pages in mdoc, update for Nvim

### PR DESCRIPTION
**Pull request by ghost**
continues #2395, refs #1826

You don't need to know mdoc to review this; just open up `nvim.1` and/or `nvimtutor.1` with `man -l`. Alternatively you can locally view the PDF files I've generated using mandoc (it can also output HTML for the website). I'll remove them before this is merged

As I said there's no need to, but if you want to learn mdoc then I recommend this guide: http://manpages.bsd.lv/
It's very comprehensive, but mdoc is quite simple so it only took a few hours.

---

@splinterofchaos I'm not sure I addressed what you said earlier (https://github.com/neovim/neovim/pull/2395#discussion_r28200533) as you intended, so please take a look.

edit: oh wow, you can apparently preview PDFs on GitHub (the text rendering isn't great though): https://github.com/Pyrohh/neovim/blob/manpages/man/nvim.pdf

### CODE REVIEW

File: man/nvim.1 [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+.It Fl -headless
+Do not start a user interface.
+.It Fl h , -help
+Print usage information and exit.
+.It Fl v , -version
+Print version information and exit.
+.El
+.Sh ENVIRONMENT
+.Bl -tag -width VIMRUNTIME
+.It Ev VIMINIT
```
Comment (Pyrohh) [inline man/nvim.1]: Some of these env variable descriptions aren't very good, probably because I don't really know what differentiates some of them. Does anyone have any improvements?

Comment (kopischke) [inline man/nvim.1]: `VIMINIT` is only used outside Vim: it contains commands to execute on Vim startup (as a literal ex command line). For `VIMRUNTIME` and `VIM`, the best references are `help $VIMRUNTIME` and `help $VIM` (in a nutshell, `VIMRUNTIME` is the application provided runtime path, while `VIM` is probably best seen a way to set a shared runtime path on a system).

File: man/nvim.1 [hunk 2]
```diff
@@ -0,0 +1,375 @@
...
+If none are specified, open an empty buffer.
+If multiple files are specified, open one buffer for each file.
+To switch between buffers, use the
+.Ic :next
+and
+.Ic :previous
+commands.
+.It Fl
+Read text from standard input until EOF, then open a buffer with that text.
+Commands are read from standard error, which should be a terminal.
```
Comment (splinterofchaos) [inline man/nvim.1]: I think TTY or PTY would be more accurate, but this sounds odd because an output stream isn't a terminal. Maybe "should connect to a terminal".

File: man/nvim.1 [hunk 3]
```diff
@@ -0,0 +1,375 @@
...
+.Ic :help quickfix .
+.It There are a number of other options:
+.It Fl -
+Interpret all further arguments as files.
+Can be used to edit files starting with a hyphen.
+.It Fl -literal
+Interpret filenames literally, that is do not expand wildcards.
+Has no effect on UNIX-like systems, where the shell expands wildcards.
+.It Fl e
+Ex mode.
```
Comment (splinterofchaos) [inline man/nvim.1]: "Start in Ex mode"?

File: man/nvim.1 [hunk 4]
```diff
@@ -0,0 +1,375 @@
...
+Improved Ex mode.
+.It Fl s
+Silent mode.
+Only takes effect if
+.Fl e
+is specified before
+.Fl s .
+.It Fl d
+Diff mode.
+Shows the difference between up to four files.
```
Comment (splinterofchaos) [inline man/nvim.1]: We don't give any example of using `-d`. Maybe just putting `[files...]` after the `-d` would suffice.

**Comment by splinterofchaos** [comment]
> @splinterofchaos I'm not sure I addressed what you said earlier (#2395 (comment)) as you intended, so please take a look.

Yeah, it's the `+[linenum]`, `+/pattern`, and `-c command`. `[linenum]` and `/pat` are both commands and one could equivalently do `-c [linenum]`, `-c /pat`, or `+command`.

File: man/nvim.1 (hunk 2 above)
Comment (Pyrohh) [inline man/nvim.1]: > I think TTY or PTY would be more accurate

I used terminal because it's more generic, and TTY stands for teletype ([among other things apparently](https://en.wikipedia.org/wiki/Teleprinter)), which is probably wrong in most modern contexts.

> but this sounds odd because an output stream isn't a terminal. Maybe "should connect to a terminal".

:+1: perhaps "... , which should be associated to a terminal."?

File: man/nvim.1 (hunk 3 above)
Comment (Pyrohh) [inline man/nvim.1]: :+1: I'll update the rest of the "mode" flags, but keep the `--help` output the same for brevity.

File: man/nvim.1 (hunk 4 above)
Comment (Pyrohh) [inline man/nvim.1]: :+1:

File: man/nvim.1 (hunk 3 above)
Comment (Pyrohh) [inline man/nvim.1]: Eh, I'm not sure actually, as this introduces a lot of redundancy.

File: man/nvim.1 (hunk 2 above)
Comment (splinterofchaos) [inline man/nvim.1]: :+1:

File: man/nvim.1 (hunk 1 above)
Comment (Pyrohh) [inline man/nvim.1]: > VIMINIT is only used outside Vim: it contains commands to execute on Vim startup (as a literal ex command line)

:+1: thanks

> (in a nutshell, VIMRUNTIME is the application provided runtime path, while VIM is probably best seen a way to set a shared runtime path on a system)

thanks, I'll look into that.

**Comment by Pyrohh** [comment]
Repushed with suggested changes (and some other ones) and regenerated the PDF.

@splinterofchaos 
> Yeah, it's the +[linenum], +/pattern, and -c command. [linenum] and /pat are both commands and one could equivalently do -c [linenum], -c /pat, or +command.

Ah, I understand. I mentioned that in addition to what was already there in https://github.com/Pyrohh/neovim/commit/20b3e79801041f15dff06508b5cd64d4396db602.

File: man/nvim.1 (hunk 1 above)
Comment (Pyrohh) [inline man/nvim.1]: I've updated this by the way, in case you want to take a look.

File: man/nvim.1 [hunk 5]
```diff
@@ -0,0 +1,388 @@
...
+.It Fl u Ar nvimrc
+Use
+.Ar nvimrc
+instead of the default of
+.Pa ~/.nvimrc .
+If
+.Ar nvimrc
+is
+.Cm NONE ,
+do not read any initialization files or environment variables.
```
Comment (fmoralesc) [inline man/nvim.1]: You need to mention `-u NORC` too, I think, which doesn't load the nvimrc file, but loads system-wide plugins.

File: man/nvim.1 (hunk 3 above)
Comment (fmoralesc) [inline man/nvim.1]: Also, add `see :help Ex-mode`.

File: man/nvim.1 [hunk 6]
```diff
@@ -0,0 +1,388 @@
...
+.It Fl -
+Interpret all further arguments as files.
+Can be used to edit files starting with a hyphen.
+.It Fl -literal
+Interpret filenames literally, that is do not expand wildcards.
+Has no effect on UNIX-like systems, where the shell expands wildcards.
+.It Fl e
+Ex mode.
+.It Fl E
+Improved Ex mode.
```
Comment (fmoralesc) [inline man/nvim.1]: Add `see :help gQ`

File: man/nvim.1 [hunk 7]
```diff
@@ -0,0 +1,388 @@
...
+.It Fl s
+Silent mode.
+Only takes effect if
+.Fl e
```
Comment (fmoralesc) [inline man/nvim.1]: It also takes effect if `-E` is specified.

File: man/nvim.1 [hunk 8]
```diff
@@ -0,0 +1,388 @@
...
+.Cm NONE ,
+do not read or write an nviminfo file.
+See
+.Ic :help viminfo .
+.It Fl -noplugin
+Skip loading plugins.
+Implied by
+.Cm -u NONE .
+.It Fl o Ns Op Ar N
+Open
```
Comment (fmoralesc) [inline man/nvim.1]: Neither this nor the help say what happens when N is less than the number of file arguments given (it gives the first N files windows or tabpages and hides the rest). Just as a heads up.

Comment (fmoralesc) [inline man/nvim.1]: Same for `-O` and `-p`.

File: man/nvim.1 [hunk 9]
```diff
@@ -0,0 +1,388 @@
...
+.Ar command
+after reading the first file.
+Up to 10 instances of
+.Fl c
+or
+.Cm +
+can be used.
+.It Fl S Op Ar session
+Source
+.Ar session
```
Comment (fmoralesc) [inline man/nvim.1]: What is a session? Maybe add `see :help session-file`

File: man/nvim.1 [hunk 10]
```diff
@@ -0,0 +1,388 @@
...
+after the first file argument has been read.
+Equivalent to
+.Cm -c \(dqsource session\(dq .
+.Ar session
+cannot start with a hyphen.
+If
+.Ar session
+is omitted, then
+.Pa Session.vim
+is used (only works if
```
Comment (fmoralesc) [inline man/nvim.1]: "is looked for and, if found, used" (phrase it better than me, obvs, though).

File: man/nvim.1 (hunk 5 above)
Comment (Pyrohh) [inline man/nvim.1]: :+1:

File: man/nvim.1 (hunk 6 above)
Comment (Pyrohh) [inline man/nvim.1]: :+1:

File: man/nvim.1 (hunk 7 above)
Comment (Pyrohh) [inline man/nvim.1]: :+1:

File: man/nvim.1 (hunk 8 above)
Comment (Pyrohh) [inline man/nvim.1]: :+1:

File: man/nvim.1 (hunk 9 above)
Comment (Pyrohh) [inline man/nvim.1]: :+1:

File: man/nvim.1 (hunk 10 above)
Comment (Pyrohh) [inline man/nvim.1]: :+1:

**Comment by Pyrohh** [comment]
@fmoralesc Thanks for the review, I pushed your changes. I'll squash everything and mark as RDY if it looks good to you (and everyone else).

**Comment by fmoralesc** [comment]
LGTM, with my last comment regarding `NORC`.

**Comment by Pyrohh** [comment]
I attempted to address your comment @fmoralesc.

**Comment by Pyrohh** [comment]
I set this to RDY; I'll wait a day or two to merge if there's no further comments.

File: src/nvim/main.c [hunk 11]
```diff
@@ -905,12 +905,7 @@ static void command_line_scan(mparm_T *parmp)
           if (STRICMP(argv[0] + argv_idx, "help") == 0)
             usage();
           else if (STRICMP(argv[0] + argv_idx, "version") == 0) {
-            Columns = 80;                 /* need to init Columns */
```
Comment (justinmk) [inline src/nvim/main.c]: why doesn't Columns need to be initialized?

File: src/nvim/main.c [hunk 12]
```diff
@@ -905,12 +905,7 @@ static void command_line_scan(mparm_T *parmp)
...
-            info_message = TRUE;           /* use mch_msg(), not mch_errmsg() */
-            list_version();
-            msg_putchar('\n');
-            msg_didout = FALSE;
-            mch_exit(0);
+            version();
```
Comment (justinmk) [inline src/nvim/main.c]: It would be better to leave the explicit mch_exit(0) in both cases, instead of hiding it in version() and having to explain what's going on.

File: src/nvim/main.c [hunk 13]
```diff
@@ -1985,41 +1992,42 @@ static void usage(void)
...
+  mch_msg(_("  -i <nviminfo>         Use <nviminfo> instead of .nviminfo\n"));
   mch_msg(_("  --noplugin            Don't load plugin scripts\n"));
-  mch_msg(_("  -p[N]                 Open N tab pages (default: one for each file)\n"));
   mch_msg(_("  -o[N]                 Open N windows (default: one for each file)\n"));
   mch_msg(_("  -O[N]                 Like -o but split vertically\n"));
+  mch_msg(_("  -p[N]                 Open N tab pages (default: one for each file)\n"));
   mch_msg(_("  +                     Start at end of file\n"));
-  mch_msg(_("  +<lnum>               Start at line <lnum>\n"));
+  mch_msg(_("  +<linenum>            Start at line <linenum>\n"));
+  mch_msg(_("  +/<pattern>           Start at first occurrence of <pattern>\n"));
```
Comment (justinmk) [inline src/nvim/main.c]: :)

File: runtime/doc/Makefile [hunk 14]
```diff
@@ -51,12 +30,6 @@ tags: doctags $(DOCS)
 doctags: doctags.c
 	$(CC) doctags.c -o doctags
 
-manpages: $(MANPAGES)
-
-# OSX groff doesn't support utf-8 as input encoding, so this won't work there.
-.1.man:
-	groff -k -mandoc -Tutf8 $< | sed -e s/.^H//g > $@
```
Comment (justinmk) [inline runtime/doc/Makefile]: why was this removed?

**Comment by justinmk** [comment]
So the reason the manpages were moved out of `runtime/doc/`, is that we don't want to ship them, just install them? I am not an advanced manpage user, but I wonder if anyone objects to that. Is there any reason a user would want the `.1` files in their `/user/share/...`?

File: src/nvim/main.c (hunk 12 above)
Comment (Pyrohh) [inline src/nvim/main.c]: :+1:

**Comment by Pyrohh** [comment]
> So the reason the manpages were moved out of runtime/doc/, is that we don't want to ship them, just install them?

Yes, pretty much. I don't think it makes sense them being in runtime, because... well they're not part of the nvim runtime.

> Is there any reason a user would want the .1 files in their /user/share/...?

If the man pages are not installed to `/usr/{share}/man/man[1-8]`, or wherever a particular systems manual viewer expects them (could be specified via `man.conf`, `MANPATH`, etc.), then typing `man nvim` won't find the manual page.

File: runtime/doc/Makefile (hunk 14 above)
Comment (Pyrohh) [inline runtime/doc/Makefile]: After seeing this[1] I wasn't sure that it's actually true. Assuming a moderately recent (last decade) version of groff, man pages written in mdoc don't need any pre-processing; you just type `man my-program` and your man page is formatted on the fly, then displayed.

Could you test how `nvim.1` and `nvimtutor.1` look on OS X? You should be able to view them using `man -l`.

[1]: https://developer.apple.com/library/mac/documentation/Darwin/Reference/ManPages/man1/groff.1.html

File: src/nvim/main.c (hunk 11 above)
Comment (Pyrohh) [inline src/nvim/main.c]: It's already done in `globals.h`.

File: src/nvim/main.c (hunk 13 above)
Comment (Pyrohh) [inline src/nvim/main.c]: Heh, I wasn't even aware it was missing until I checked the man page :grin:

File: runtime/doc/Makefile (hunk 14 above)
Comment (justinmk) [inline runtime/doc/Makefile]: It looks fantastic actually.

On Mac (BSD I guess), there's no `man -l`, but a slash in the path works similarly: `man ./nvim.1`.

**Comment by Pyrohh** [comment]
@justinmk Thanks for taking a look, repushed with your suggestions, and fixed up that commit message.

File: man/nvim.1 [hunk 15]
```diff
@@ -0,0 +1,417 @@
...
+but truncate
+.Ar scriptout .
+.It Fl -startuptime Ar file
+During startup, append timing messages to
+.Ar file .
+Can be used to diagnose slow startup times.
+.It Fl -api-info
+Dump API metadata serialized to msgpack and exit.
+.It Fl -embed
+Use standard input and standard output as a msgpack-rpc channel.
```
Comment (justinmk) [inline man/nvim.1]: stdin and stdout

File: man/nvim.1 [hunk 16]
```diff
@@ -0,0 +1,417 @@
...
+If none are specified, open an empty buffer.
+If multiple files are specified, open one buffer for each file.
+To switch between buffers, use the
+.Ic :next
+and
+.Ic :previous
+commands.
+.It Fl
+Read text from standard input until EOF, then open a buffer with that text.
+Commands are read from standard error, which should be associated with a
```
Comment (justinmk) [inline man/nvim.1]: I find it very offputting to spell out stdin and stderr. The *only* people for whom stdin/out/err is useful are people who will actually be *more confused* by the pronouncement "standard input". People who aren't used to these concepts won't be helped by the non-abbreviated form.

File: man/nvim.1 (hunk 15 above)
Comment (Pyrohh) [inline man/nvim.1]: I had it that way before, but changed it since they aren't being talked about in programming contexts (i.e., the actual file descriptors).

File: man/nvim.1 [hunk 17]
```diff
@@ -0,0 +1,417 @@
...
+terminal.
```
Comment (justinmk) [inline man/nvim.1]: I also think it's a hyper-correction to change this from the original wording, 

> Commands are read from stderr, which should be a tty.

The new form is totally meaningless to me. I don't know what "associated with a terminal" means, but I immediately understand "should be a tty". 

TTY is technical jargon with a specific meaning that is understood in context. No one in the context of a manpage mistakes it for physical teletypes.

File: man/nvim.1 [hunk 18]
```diff
@@ -0,0 +1,417 @@
...
+If
+.Ar errorfile
+is omitted, the value of the 'errorfile' option is used (defaults to
+.Cm errors.err ) .
+Further errors can be jumped to with the
+.Ic :cnext
+command.
+See
+.Ic :help quickfix .
+.It There are a number of other options:
```
Comment (justinmk) [inline man/nvim.1]: Can just say "Other options:"

File: man/nvim.1 [hunk 19]
```diff
@@ -0,0 +1,417 @@
...
+Most of
+.Xr vim 1
+was written by
+.An -nosplit
+.An Bram Moolenaar ,
+with a lot of help from others.
+See
+.Ic :help credits .
+.Pp
+Although hardly any of the original code remains,
```
Comment (justinmk) [inline man/nvim.1]: can just delete this sentence.

File: man/nvim.1 [hunk 20]
```diff
@@ -0,0 +1,417 @@
...
+.It Ev VIMRUNTIME
+Used to locate run time files, such as on-line documentation and
+syntax highlighting definitions.
+.It Ev SHELL
+Used to set the 'shell' option, which determines the shell used by the
+.Ic :terminal
+command.
+.It Ev NVIM_TUI_CURSOR_SHAPE
+If set to
+.Cm 1 ,
```
Comment (justinmk) [inline man/nvim.1]: non-zero

File: man/nvim.1 [hunk 21]
```diff
@@ -0,0 +1,417 @@
...
+change the cursor shape to a vertical bar while in insert mode.
+Requires that the host terminal support the DECSCUSR CSI escape sequence.
+Has no effect in GUIs.
+.It Ev NVIM_TUI_TRUE_COLOR
+If set to
+.Cm 1 ,
```
Comment (justinmk) [inline man/nvim.1]: `If non-zero`

File: man/nvim.1 (hunk 17 above)
Comment (Pyrohh) [inline man/nvim.1]: How is it totally meaningless? Perhaps `assosciated` should be `refers to`, but besides that I don't understand how it's totally meaningless. If anything, TTY, besides being a misnomer, is less informative to users unaware of the term, where as "terminal" is self-explanatory. Someone might see "should be a terminal" and say "oh, well I'm running a terminal so I'm fine".

Comment (justinmk) [inline man/nvim.1]: I don't get the insistence on TTY being a misnomer. Literally *everything* in computing is an analogy of some physical concept. TTY is a technical term with a long history in unix, as is "socket", "port", "core dump", "page", ...

It makes no sense to try to make it friendly, it won't help any user (who probably also doesn't know the difference between a terminal and a shell--not to mention a pseudo-terminal...).

> Someone might see "should be a terminal" and say "oh, well I'm running a terminal so I'm fine".

So they would then know how to send commands to `stderr`? That's quite optimistic :)

File: man/nvim.1 (hunk 16 above)
Comment (Pyrohh) [inline man/nvim.1]: > The only people for whom stdin/out/err is useful are people who will actually be more confused by the pronouncement "standard input"

stdXXX are for programming contexts, where as standard XXX are not (this seems to be the trend in the majority of manual pages I looked at, GNU and BSD), as the former refers to the file descriptor. I don't know about others, but I don't see why people would be more confused: the non-abbreviated version is more correct in that it's a general descriptiong

Besides that, I don't see why we should deviate what seems to be the de facto standard in man page language.

File: man/nvim.1 (hunk 19 above)
Comment (Pyrohh) [inline man/nvim.1]: :+1:

File: man/nvim.1 (hunk 20 above)
Comment (Pyrohh) [inline man/nvim.1]: :+1:  `If defined` is good currently. I have a PR for that at #2569.

File: man/nvim.1 (hunk 18 above)
Comment (Pyrohh) [inline man/nvim.1]: I didn't do that because the above `-t` and `-q` are also options. Given that the DESCRIPTION has shrunk substantially, I just merged the two sections.

Comment (justinmk) [inline man/nvim.1]: There really should be an options section. It's ok if description section contains the  options above this line.

File: man/nvim.1 (hunk 16 above)
Comment (justinmk) [inline man/nvim.1]: `man cat` agrees with you. Meh.

File: runtime/doc/Makefile (hunk 14 above)
Comment (Pyrohh) [inline runtime/doc/Makefile]: Thanks for checking, that's great to hear (I was afraid we'd have to do some magic to it).

File: man/nvim.1 (hunk 17 above)
Comment (Pyrohh) [inline man/nvim.1]: > I don't get the insistence on TTY being a misnomer. Literally everything in computing is an analogy of some physical concept. TTY is a technical term with a long history in unix, as is "socket", "port", "core dump", "page"

I hadn't thought about that, although It's still a misnomer in that it's usually not used in the context of Teleprinters, where are terminal makes more sense here because it's just a front-end to a system (like an interactive shell is to a login shell). It doesn't really matter much.

> It makes no sense to try to make it friendly, it won't help any user (who probably also doesn't know the difference between a terminal and a shell--not to mention a pseudo-terminal...).

I suppose I wasn't clear, but I'm more concerned about it being correct than friendly (not that they're mutually exclusive). And since "correct" isn't so easily defined in this context, I mean following the de facto standards regarding man pages.

The main point I'm making is that `tty` is to `terminal` as `stdXXX` is to `standard XXX`, at least from what I've gathered from various man pages. `tty` looks to be used almost exclusively in a non-general way, things like function names or `/dev` paths, which is no surprise given how UNIX influenced POSIX. However, even POSIX's [`man isatty`](http://pubs.opengroup.org/onlinepubs/009695399/functions/isatty.html) (among many others) uses the term "terminals", as there's no need to continue to use a misnomer just because the name is used. Most man pages I've read agree as well.

File: man/nvim.1 (hunk 18 above)
Comment (Pyrohh) [inline man/nvim.1]: I don't think we should add an options section for the sake of adding it, especially since it makes it inconsistent. It's pretty clear where the options start due to the indentation. The use of an options section is inconsistent among manuals, and doesn't really make sense here given how small the description is.

File: man/nvim.1 [hunk 22]
```diff
@@ -0,0 +1,411 @@
...
+inside
+.Nm .
+.It Cm +/ Ns Op Ar pattern
+For the first file, position the cursor on the first occurrence of
+.Ar pattern .
+If
+.Ar pattern
+is omitted, the most recently used search pattern is used (if there is one).
+Note that
+.Cm /foo
```
Comment (yagebu) [inline man/nvim.1]: `+/foo`

File: man/nvim.1 [hunk 23]
```diff
@@ -0,0 +1,411 @@
...
+See
+.Ic :help search-pattern .
+.It Fl -cmd Ar command
+Execute
+.Ar command
+before processing any nvimrc.
+Up to 10 instances of these can be used, independently from instances of
+.Fl c .
+.It Fl c Ar command
+Execute
```
Comment (yagebu) [inline man/nvim.1]: It should be explained that `+command` works the same way as `-c command`, so maybe add `+command` in the left column. From the lines above it might seem that the '+' only works with linenumbers and search patterns

File: man/nvim.1 (hunk 22 above)
Comment (Pyrohh) [inline man/nvim.1]: Nice catch, fixed.

File: man/nvim.1 (hunk 23 above)
Comment (Pyrohh) [inline man/nvim.1]: Huh, good point. I really don't like how similar all these commands are...

Comment (Pyrohh) [inline man/nvim.1]: Actually, I'm not sure... it's already like that in the existing documentation (although that's not much of a reason in itself, as we can just change the documentation).

At least for now, it might be easier to leave it, where it says that `+` should be used for line numbers, and the other ones (`+/`, `-c`, `--cmd`) should be used for other purposes, as opposed to saying something like `Here's four commands which are mostly the same but not really` (e.g., you can't do `+'e file'` but you can do `-c 'e file'`.

I think this is something that could be improved in the future, but I'm not sure about now. What do you think?

Comment (yagebu) [inline man/nvim.1]: > it's already like that in the existing documentation

No ;) Actually `+command` is even listed first and it's also explained how you can use it with commands that include spaces (the '+' has to be between the quotes too):
  
    +{command}
    .TP
    \-c {command}
    {command} will be executed after the
    first file has been read.
    {command} is interpreted as an Ex command.
    If the {command} contains spaces it must be enclosed in double quotes (this
    depends on the shell that is used).
    Example: Vim "+set si" main.c

I definitely agree that it's not ideal to have all these very similar options, but as `+command` is the most general one and `+num` and `+/pat` are just special cases of it, I really think we should keep it.

Comment (yagebu) [inline man/nvim.1]: > +num and +/pat are just special cases of it

To be exact, the one exception is that just a `+` will put the cursor at the end of the file.

Comment (Pyrohh) [inline man/nvim.1]: Oops, well thanks for clearing that up. I'll update it and ping you after.

**Comment by Pyrohh** [comment]
@yagebu I attempted to modify the man page per your suggestion, what do you think?

**Comment by Pyrohh** [comment]
It doesn't appear here, but the most recent suggestion was discussed here:
https://github.com/Pyrohh/neovim/commit/ba406b5c865a3c947b9a3fcec07764fee51c9245#commitcomment-11170474.

I'll merge after the tests.

**Comment by Pyrohh** [comment]
One of the builds has stalled, but everything else passed so I'm merging this.
